catalyst-gateway/tests/api_tests/api/v1/document.py

```diff
@@ -1,27 +1,42 @@
 import requests
 from api import cat_gateway_endpoint_url
 
-URL = cat_gateway_endpoint_url("api/v1/document")
```

Comment:
Pls restore this line, it makes more clear which endpoint we are testing and avoids repetitions